pandas/tseries/converter.py

```diff
@@ -172,7 +172,7 @@ def _dt_to_float_ordinal(dt):
     is a :func:`float`.
     """
     if (isinstance(dt, (np.ndarray, Index, Series)
-                   ) and is_datetime64_ns_dtype(dt)):
```

Comment:
Actually, we already have this:

```
        rs = self.dtc.convert(np.array([
            np_datetime64_compat('2012-01-01 00:00:00+0000'),
            np_datetime64_compat('2012-01-02 00:00:00+0000')]), None, None)
        self.assertEqual(rs[0], xp)
```

does already test this as the above has [s] resolution, not [ns] .

Would just add a test case for the tz case.